src/main/metrics/push-gateway.ts

```diff
@@ -0,0 +1,41 @@
+import { Request } from '@automationcloud/request';
+import { inject, injectable, multiInject } from 'inversify';
+
+import { Config, config } from '../config';
+import { Logger } from '../logger';
+import { MetricsRegistry } from './registry';
+
+const DEFAULT_PUSH_GATE_URL = 'http://push-gateway.monitoring.svc.cluster.local:9091/metrics';
```

Comment:
Let's either inline tit or rename to `DEFAULT_PUSH_GATEWAY_URL` (though I'd prefer inlining and making the `@config` multi-line)